src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -42,6 +46,24 @@ def api_error(self, error, action_type, logging_data):
         logging_data["action_type"] = action_type
         logger.info("slack.action.api-error-pre-message: %s" % six.text_type(logging_data))
         logger.info("slack.action.api-error", extra=logging_data)
+
+        if error.status_code == 403:
+            associate_url = build_unlinking_url(
+                logging_data["integration_id"],
```

Comment:
@NisanthanNanthakumar We should not use logging data like this to drive core application logic. It's not how someone should reasonably expect this parameter to be used. I recommend finding a different way of handling this that does not require reading from `logging_data`.